src/main/kotlin/com/toasttab/pulseman/AppState.kt

```diff
@@ -24,78 +24,87 @@ import com.toasttab.pulseman.entities.ClassInfo
 import com.toasttab.pulseman.entities.ProjectSettingsV1
 import com.toasttab.pulseman.entities.ProjectSettingsV2
 import com.toasttab.pulseman.entities.ProjectSettingsV3
-import com.toasttab.pulseman.entities.TabValuesV3
 import com.toasttab.pulseman.files.FileManagement
 import com.toasttab.pulseman.jars.JarManager
 import com.toasttab.pulseman.jars.LoadedClasses
+import com.toasttab.pulseman.jars.RunTimeJarLoader
+import com.toasttab.pulseman.jars.TabJarManager
 import com.toasttab.pulseman.pulsar.filters.AuthClassFilter
-import com.toasttab.pulseman.pulsar.filters.protobuf.GeneratedMessageV3Filter
-import com.toasttab.pulseman.pulsar.filters.protobuf.KTMessageFilter
 import com.toasttab.pulseman.pulsar.handlers.PulsarAuthHandler
-import com.toasttab.pulseman.pulsar.handlers.PulsarMessageClassInfo
 import com.toasttab.pulseman.state.GlobalFeedback
 import com.toasttab.pulseman.state.TabHolder
 
 class AppState {
     val globalFeedback = GlobalFeedback()
 
-    val pulsarMessageJars: JarManager<PulsarMessageClassInfo> = JarManager(
```

Comment:
Refactoring all the jar loaders so we can have a unique jarloader per tab that inherits the jars from the common and auth jar loader